README.md

```diff
@@ -19,211 +17,137 @@ Via Composer
-
-### Get connected properties
-``` php
-$propertyResource = $katanox->getPropertyResource();
-
-$properties = $propertyResource->getProperties();
-```
-These are the properties you are connected with.
-
-### Get a property
-``` php
-$propertyResource = $katanox->getPropertyResource();
-
-$property= $propertyResource->getProperty('<PROPERTY_ID>');
-```
-
-### Get a property 
-``` php
-$propertyResource = $katanox->getPropertyResource();
-
-$property= $propertyResource->getProperty('<PROPERTY_ID>');
-```
-
-### Get a property unit
-``` php
-$propertyResource = $katanox->getPropertyResource();
-
-$unit = $propertyResource->getUnit('<PROPERTY_ID>', '<UNIT_ID>');
-```
-
-### Get a property rate plan
-``` php
-$propertyResource = $katanox->getPropertyResource();
-
-$ratePlan = $propertyResource->getRatePlan('<PROPERTY_ID>', '<RATE_PLAN_ID>');
-```
-
-## Change log
-
-Please see [CHANGELOG](CHANGELOG.md) for more information on what has changed recently.
-
-## Testing
-
-``` bash
-$ composer test
-```
-
-## Contributing
-
-Please see [CONTRIBUTING](CONTRIBUTING.md) for details.
-
-## Security
-
-If you discover any security related issues, please email the Katanox Engineering Team (developers@katanox.com) instead of using the issue tracker.
-
-## License
-
-The MIT License (MIT). Please see [License File](LICENSE.md) for more information.
+### Manual Installation
+
+Download the files and include `autoload.php`:
+
+```php
+<?php
+require_once('/path/to/OpenAPIClient-php/vendor/autoload.php');
+```
+
+## Getting Started
+
+$apiInstance = new Katanox\Api\AvailabilityApi(
+    // If you want use custom http client, pass your client which implements `GuzzleHttp\ClientInterface`.
+    // This is optional, `GuzzleHttp\Client` will be used as default.
+    new GuzzleHttp\Client()
+);
+$check_in = 'check_in_example'; // string | Date(YYYY-MM-DD)
+$check_out = 'check_out_example'; // string | Date(YYYY-MM-DD)
+$authorization = 'authorization_example'; // string | Type 'Bearer' and then your API Token
+$adults = 1; // int | Number of adults
+$children = 0; // int | Number of children
+$lat = 3.4; // float | The Latitude
+$lng = 3.4; // float | The Longitude
+$radius = 2000; // int | The search radius in meters(m)
+$property_ids = array('property_ids_example'); // string[] | List of property IDs to be included. When specified, only these properties will be included in the response. The maximum size of property id list is 50.
+$negotiated_rate_plans = array('negotiated_rate_plans_example'); // string[] | List of negotiated rate plan ids to be included.
+$number_of_units = 1; // int | The total number of units required
+$page = 0; // int | The returned page number
+$limit = 10; // int | Number of results per page. The maximum value of the limit is 50.
+
+try {
+    $result = $apiInstance->getAvailableProperties($check_in, $check_out, $authorization, $adults, $children, $lat, $lng, $radius, $property_ids, $negotiated_rate_plans, $number_of_units, $page, $limit);
```

Comment:
This is actually not correct. We should either pass a lat/lng or property_ids

Reply:
not supported yet https://github.com/OAI/OpenAPI-Specification/issues/256

Reply:
Interesting, it seems you're right, sorry about that. 

How shall we proceed? Shouldn't we add this to the swaggerfile documentation so it will also be included as output here?

Reply:
We can add it manually here, but the docs/availabilityapi.md file contains a generated duplicate of this paragraph. The problem with adding it manually is that if you generate the output it will be overwritten. You can ignore it but it will not be kept in sync if you, say, adjust the specification and regenerate the output.